browser_patches/chromium/build.sh

```diff
@@ -107,14 +107,18 @@ compile_chromium() {
       /c/Windows/System32/cmd.exe "/c $(cygpath -w "${SCRIPT_FOLDER}"/buildwingoma.bat)"
     fi
   else
-    gn gen out/Default
     if [[ -z "$TARGETS" ]]; then
       if [[ $(uname) == "Linux" ]]; then
         TARGETS="chrome chrome_sandbox clear_key_cdm"
       else
         TARGETS="chrome"
       fi
     fi
+    echo
+    echo ">> Compiling Targets: $TARGETS"
+    echo
+
+    gn gen out/Default
```

Comment:
does this fix chromium with symbols on macos?

Reply:
yep should